resources/references/security.md

```diff
@@ -4,10 +4,34 @@
 
 This reference presents a comprehensive compilation of all security measures implemented in Shopware 6, along with instructions on how to configure them.
 
+## ACL in the Administration
+
+The Access Control List (ACL) in Shopware ensures that by default, data can only be created, read, updated, or deleted (CRUD), once the user has specific privileges for a module. [ACL in the Administration](../../concepts/framework/architecture/administration-concept.md#acl-in-the-administration)
+
 ## API aware fields
 
 The `ApiAware` flag allows you to control what fields of your entity are exposed to the Store API. For more information, refer to [Flags Reference](core-reference/dal-reference/flags-reference.md).
 
+## Captchas
+
+Captchas helps verify the user's humanity and prevents automated bots or scripts from gaining access. For more information, refer to [Captchas](https://docs.shopware.com/en/shopware-en/settings/basic-information#captcha)
+
+## CSP
+
+[Content Security Policies](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) are used to prevent Cross-Site-Scripting (XSS) attacks, as well as data injection attacks. The content security policy specify from which sources additional content (e.g. images, scripts, ...) can be included.
```

Comment:
```suggestion
[Content Security Policies](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) (CSPs) are used to prevent Cross-Site-Scripting (XSS) attacks, as well as data injection attacks. This policy specifies the sources from which additional content (e.g., images, scripts, etc.) can be included.
```